CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required (VERSION 3.2)
+project (lightdm-mini-greeter
+    VERSION 0.2.0)
+
+# Set compiler flags
+set(CMAKE_C_FLAGS "-march=native -mtune=native -O3 --std=c11")
+
+# Use the package PkgConfig to find libraries and headers
+find_package(PkgConfig REQUIRED)
+
+# Detect GTK+ headers/library files
+pkg_check_modules(GTK3 REQUIRED gtk+-3.0)
+
+# Find lightdm library and headers
+pkg_check_modules(LIGHTDM REQUIRED liblightdm-gobject-1>=1.2.1)
```

Comment:
This was `>=1.12` in autotools. Does `v1.2.1` actually work with the greeter?

Reply:
I think i misread the version, i'll update it to >=1.12.

Reply:
Actually bump it to `>=1.20`, #13 shows where this breaks between `1.18` & `1.20`(I couldn't find a `v.19` on their [launchpad](https://code.launchpad.net/lightdm/+code?field.lifecycle=MATURE&field.lifecycle-empty-marker=1&field.sort_by=by+most+interesting&field.sort_by-empty-marker=1)).